src/main/kotlin/com/hypto/iam/server/apis/KeyApi.kt

```diff
@@ -0,0 +1,39 @@
+package com.hypto.iam.server.apis
+
+import com.google.gson.Gson
+import com.hypto.iam.server.models.PublicKeyResponse
+import com.hypto.iam.server.service.MasterKey
+import io.ktor.application.call
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respondText
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import java.util.Base64
+import org.koin.ktor.ext.inject
+
+fun Route.keyApi() {
+    val gson: Gson by inject()
+
+    get("/public_keys/{kid}") {
```

Comment:
Named the url as public_keys/{kid} rather than keys/{kid} , as for private keys api (if we need in future) we need to create a new api endpoint as that would be protected endpoint (with some permission) and this endpoint will be permission less.